internal/version.go

```diff
@@ -0,0 +1,4 @@
+package internal
+
+// Version of executables.
+var Version = "0.1"
```

Comment:
Or, maybe start at "0.0.0+git" and then the commit that we tag and release will bump the version?